src/style.css

```diff
@@ -0,0 +1,80 @@
+@font-face {
+  font-family: Avenir;
+  src: url("./fonts/Avenir.ttc");
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  src: url("./fonts/Avenir-Book.ttf");
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 900;
+  src: url("./fonts/Avenir-Heavy.ttf");
+}
+
+* {
```

Comment:
better for html

Reply:
And reset to default browser settings?
* {
  all: unset;
  display: revert;
  box-sizing: border-box;
}